### ISSUE TITLE
Google Material charts don't work

### ISSUE THREAD

**Issue by stechnique** [wkhtmltopdf/wkhtmltopdf#2571]
I just spent a few hours trying to get material line charts to show up on my PDF and I have come to the conclusion that it does not currently work with wkhtmltopdf.
I am on version 0.12.2.1 (with patched qt) on OSX (64-bit).

I have followed pointers to --enable-javascript and added a --javascript-delay, but this:

    google.load('visualization', '1.1', {packages: ['line'], callback: 'drawChart'});
    [... later inside the drawChart function]
    var chart = new google.charts.Line(document.getElementById("pump_chart"));
    chart.draw(data, options);

Works fine in Chrome, but once turned into a PDF the chart is replaced with a red rectangle that says "'undefined' is not a function".
Keeping the same code but reverting to the standard (non-Material) line charts:

    google.load('visualization', '1.1', {packages: ['corechart'], callback: 'drawChart'});
    [...]
    var chart = new google.visualization.LineChart(document.getElementById("pump_chart"));
    chart.draw(data, options);

I'm not sure this can be fixed on your side, but I'm still leaving this here in case someone else runs into it.

**Comment by ratbeard** [comment]
Several months ago I ran in to the same issue that material charts would not work but standard charts would.  I just tried it again this morning and I'm able to render material charts now, at least a basic donut and bar chart.  I'm using `wkhtmltopdf-0.9.9-OS-X.i368`

My guess is some parts of the material charts library use Function.prototype.bind which isn't in wkhtmltopdf.  Dropping in a polyfill for it might fix your error.

**Comment by gitamitgit** [comment]
Facing same issue , while converting HTML, google charts showing errors , showing blank rectangles instead of chart

**Comment by Tomsgu** [comment]
Is this still an issue in 0.12.5? What does it say if you add `--debug-javascript` option?

**Comment by mksowmya** [comment]
Yes, this still an error on 0.12.5. Has anybody solved this issue?
